src/components/Tabbar.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Paper from '@material-ui/core/Paper';
+import AlbumOutlinedIcon from '@material-ui/icons/AlbumOutlined';
+import AccountBalanceWalletOutlinedIcon from '@material-ui/icons/AccountBalanceWalletOutlined';
+import Mood from '@material-ui/icons/Mood';
+
+const styles = {
+  root: {
+    flexGrow: 1,
+    boxShadow: 'none'
+  },
+};
+
+class Tabbar extends Component {
+  state = {
+    value: 0,
+  };
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { value } = this.state;
+
+    return (
+      <Paper className={classes.root}>
+        <Tabs
```

Comment:
I think the actual tabs need to be `space-evenly` or `space-between` to be aligned with the design in Figma

Reply:
Done